spec/requests/bookmarks_controller_spec.rb

```diff
@@ -142,4 +144,62 @@
       end
     end
   end
+
+  describe "#bulk" do
+    it "needs you to be logged in" do
+      put "/bookmarks/bulk.json"
+      expect(response.status).to eq(403)
+    end
+
+    describe "when logged in" do
+      before { sign_in(current_user) }
+
+      let!(:operation) { { type: "clear_reminder" } }
+      let!(:bookmark_ids) { [1, 2] }
+
+      it "requires a list of bookmark_ids" do
+        put "/bookmarks/bulk.json", params: { operation: operation }
+        expect(response.status).to eq(400)
+      end
+
+      it "requires an operation param" do
+        put "/bookmarks/bulk.json", params: { bookmark_ids: bookmark_ids }
+        expect(response.status).to eq(400)
+      end
+
+      it "can clear reminder for the given bookmarks" do
+        Guardian.any_instance.stubs(:can_edit?).returns(true)
```

Comment:
Instead of stubbing the guardian check, it might be better to run this call as the correct user. And also run it as a second user (which should fail to update the bookmarks for user 1.)